### DISCUSSION TITLE
The number of protein isoforms identified at 1% FDR

### DISCUSSION THREAD

**Discussion by starry1998** [vdemichev/DiaNN#218] [Q&A]
Hi Vadim,
I would like to know how the number of protein isoforms identified is calculated in the report.log.txt?
I saw this message here #180 .
"length(unique(df$Genes[df$Proteotypic == 1 & df$Protein.Q.Value <= 0.01]))"
but the column of Genes and ProteinName is empty in my report.tsv.
could you give me any information about this number.

Best,
Ying

**Comment by vdemichev** [comment]
Hi Ying, that message was about calculating the number of Genes. Isoforms are UniprotIDs and are listed in the Protein.Ids column.

Best,
Vadim
